src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/GjennoptaBehandling/GjennoptaBehandling.tsx

```diff
@@ -45,7 +43,6 @@ const GjennoptaBehandling: React.FC<IProps> = ({ behandling, onListElementClick
                     portal={true}
                     width="small"
                     onClose={() => {
-                        nullstillSkjema();
```

Comment:
Sikker på at dette er greit å fjerne? Hva skjer når vi åpner skjemaet på nytt?

Reply:
Denne modalen kan ikke endre på skjemaet, så jeg skjønner ikke hvorfor nullstillingen var der i utgangspunktet. Modalen er egentlig bare bestående av to knapper: "ok"-knapp som gjør at man tar behandlingen av vent og en "avbryt"-knapp som lukker modalen. Når man trykker på "avbryt"-knappen kan man ikke ha gjort noen endringer på skjemaet og det skal ikke være nødvendig med noen nullstilling